bin/collator/src/rpc.rs

```diff
@@ -319,6 +305,14 @@ where
 
     let no_tx_converter: Option<fp_rpc::NoTransactionConverter> = None;
 
+    if !client
+        .runtime_api()
+        .has_api::<dyn AuraApi<Block, AuraId>>(client.info().best_hash)
+        .unwrap_or_default()
+    {
+        return Err("EVM RPC cannot be enable at the current state. Please sync at least 200K blocks before enabling it.".into());
+    }
+
```

Comment:
So we disable EVM RPC until 200K blocks in shiden?

Reply:
Slot duration isn't available in first 200K blocks in Shiden. So you can't enable it during 200K blocks syncing.